packages/sycamore-macro/src/component.rs

```diff
@@ -307,10 +368,12 @@ fn inline_props_impl(item: &mut ItemFn) -> Result<TokenStream> {
 
     let doc_comment = format!("Props for [`{}`].", item.sig.ident);
 
+    let attrs = attrs.into_iter().collect::<Vec<_>>();
```

Comment:
I don’t think you need this line since quote! can take iterators directly already.

Reply:
it has to be there because before this line attrs is Punctuated<_, Token![,]> so the macro would otherwise produce #[derive(Clone)]**,**#[derive(Debug)] with a , in between which would result in an error